README.md

```diff
@@ -51,6 +51,70 @@ pandas-specific things that are right now hard-coded in the theme. We also need
 to work on better configurability and extensibility. Feedback and contributions
 are very welcome!
 
+## Theme development
+
+The theme is bundled via Webpack. In `./src/*` the theme related stylesheets and javascript is located. 2 entrypoints are available:
+
+- Stylesheet: `./src/scss/theme.scss'
```

Comment:
Would it make sense to factor this out into some SCSS modules and then stitch them together with a `master.scss` file that imports them? I've seen that as a common pattern for managing scss.

Reply:
I would surely make sense. You mean to rename `theme.scss` as the main entrypoint and split up styles in separate modules which are imported in `master.scss?`

Often the naming convention would be `index.scss` or `index.js` but maybe `master.*` is more clear?

Reply:
Yep, that's what I was thinking. Index would be totally fine w me